java-client/build.gradle.kts

```diff
@@ -143,10 +143,6 @@ dependencies {
     // https://search.maven.org/artifact/com.google.code.findbugs/jsr305
     api("com.google.code.findbugs:jsr305:3.0.2")
 
-    // EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
-    // https://github.com/eclipse-ee4j/jsonp
-    api("jakarta.json:jakarta.json-api:2.0.1")
-
```

Comment:
Can we remove strong dependencies of 
```
    api("com.google.code.findbugs:jsr305:3.0.2")

    api("org.eclipse.parsson:parsson:1.1.1")

    implementation("jakarta.json.bind", "jakarta.json.bind-api", "2.0.0")
```
 
   as a part of this PR?

Reply:
All of them look like they are being used.